src/platform/graphics/constants.js

```diff
@@ -946,8 +946,95 @@ export const PIXELFORMAT_R8 = 52;
  */
 export const PIXELFORMAT_RG8 = 53;
 
-// map of engine PIXELFORMAT_*** enums to information about the format
-// .srgb: true if sampling the format returns value in sRGB space, otherwise in linear space
+/**
+ * Format equivalent to {@link PIXELFORMAT_DXT1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_DXT1_SRGB = 54;
```

Comment:
Probably want to add support for SRGB8_ALPHA8  for the very common case of RGBA8 textures?

Reply:
we have PIXELFORMAT_SRGBA8, is this what you mean?